libs/remix-ui/run-tab/src/lib/components/value.tsx

```diff
@@ -58,7 +58,7 @@ export function ValueUI (props: ValueProps) {
           className="form-control udapp_gasNval udapp_col2"
           id="value"
           data-id="dandrValue"
-          title="Enter the value and choose the unit"
+          title="Enter an amount and choose its unit"
```

Comment:
It can be `Enter a value to send with tx`

Reply:
Same as above.  I think value is too general a concept.